OricExplorer/Forms/frmEditTape.Designer.cs

```diff
@@ -247,6 +247,7 @@ private void InitializeComponent()
             // txtProgramName
             // 
             this.txtProgramName.Location = new System.Drawing.Point(106, 24);
+            this.txtProgramName.MaxLength = 16;
```

Comment:
Indeed, the maximum program name on a cassette tape is 16 characters - see book "L'Oric A Nu" by Fabrice Broche p. 61 and pp. 257-258.
See also https://forum.defence-force.org/viewtopic.php?p=13821#p13821 for a discussion on mistakes on this book including errors related to tape header format.